### PR TITLE
XW-305

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-305

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
All test are passing.
DON'T MERGE IT BEFORE 17-SEP!!!

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/AllTagsTests.java [hunk 1]
```diff
@@ -37,7 +45,9 @@ public void prepareTest() {
     widgets.add(new TwitterWidgetPageObject(driver));
     widgets.add(new VKWidgetPageObject(driver));
     widgets.add(new WeiboWidgetPageObject(driver));
-    widgets.add(new GoogleFormWidgetPageObject(driver));
+
+    // Uncomment after 22-sep
+    //widgets.add(new GoogleFormWidgetPageObject(driver));
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/AllTagsTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java [hunk 2]
```diff
@@ -33,7 +40,9 @@ public void prepareTest() {
     widgets.add(new TwitterWidgetPageObject(driver));
     widgets.add(new VKWidgetPageObject(driver));
     widgets.add(new WeiboWidgetPageObject(driver));
-    widgets.add(new GoogleFormWidgetPageObject(driver));
+
+    // Uncomment after 22-sep
+    //widgets.add(new GoogleFormWidgetPageObject(driver));
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 4 major
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/fe3e3e5069e05a1783b2c61c618da168ea0fc861/src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/AllTagsTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/fe3e3e5069e05a1783b2c61c618da168ea0fc861/src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/SpotifyTests.java#L31) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/fe3e3e5069e05a1783b2c61c618da168ea0fc861/src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by tomasznapieralski** [comment]
All tests are passing on Jenkins for both Mercury and Oasis

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 4 major
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/fe3e3e5069e05a1783b2c61c618da168ea0fc861/src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/AllTagsTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/fe3e3e5069e05a1783b2c61c618da168ea0fc861/src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/SpotifyTests.java#L31) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/fe3e3e5069e05a1783b2c61c618da168ea0fc861/src/test/java/com/wikia/webdriver/testcases/widgettests/AllTagsTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Pull request merged**